integration-test/src/test/scala/com/cloudera/livy/test/JobApiIT.scala

```diff
@@ -157,16 +157,17 @@ class JobApiIT extends BaseIntegrationTestSuite with BeforeAndAfterAll with Logg
     assert(result === "hello")
   }
 
-  test("run scala jobs") {
+  test("run echo jobs") {
```

Comment:
You are right. I missed that.
Besides ScalaEcho, all other test jobs are Java classes. I think we can load them from test-lib-2.10.jar.
For ScalaEcho test, I don't have a solution yet... Let me sleep on it and get back to you.